class1.md

```diff
@@ -0,0 +1,261 @@
+
+**[Nature Genetics Review:](https://www.nature.com/articles/nrg3920)** The field of machine learning includes the development and application of computer algorithms that improve with experience.
+
+While these definitions vary slightly there are a few common themes to pull out:
+
+* Machine learning is a kind of algorithm.
+  * Algorithm - A finite sequence of well-defined, computer-implementable instructions, typically to solve a class of problems or to perform a computation 
+   * Essentially it is a protocol for a computer
+* Machine learning incorporates data into a statistical model.
+* Machine learning improves with experience without human intervention.
+
+### Machine learning as code
+
+When we think of machine learning and how it applies to code we can consider machine learning to be a specific way of programming computers.
+
+* Most programming isn’t built to learn - it’s procedural
+    * This is where a programmer builds out a set of rules. This set of rules is called an algorithm. This ruleset remains static unless the coder intervenes and specifies more rules. 
+* In machine learning the algorithm is also designed and coded by a human. However, this kind of algorithm is able to learn from the data about specific parameters that will shape a statistical model for making predictions and inferences. The coder doesn’t necessarily know or control these parameters - the machine does.
+
+> **Machine learning is an algorithm (or set of rules) that self adjusts based on the large datasets that it is given**
+
+#### Challenge!
+
+1. Write an algorithm/protocol/ruleset for making a PBJ sandwich
+2. Write an algorithm/protocol/ruleset for determining if an email is spam
+
+[click here to go to solutions page](https://github.com/fredhutchio/concepts_machine_learning/blob/master/solutions/class1_solutions.md#write-an-algorithmprotocolruleset-for-making-a-pbj-sandwich)
+
+### When to use machine learning
+
+Just like any other statistical method there are specific situations where applying machine learning is appropriate. Just as important as the algorithm itself is a solid understanding of experimental design, domain expertise in the problem area youre trying to solve, and an understanding of data governance and ethics.
+
+**You cannot code the rules:**
+* Tasks that cannot be solved using a simple, rule-based solution. 
+* When rules depend on too many factors and many of these rules overlap or need to be tuned very finely, it soon becomes difficult for a human to accurately code the rules. You can use ML to effectively solve this problem.
+* This is true for drawing conclusions about highly dimensional data.
+
+**You cannot scale:**
+* You might be able to manually recognize a few hundred emails and decide whether they are spam or not. However, this task becomes tedious for millions of emails. ML solutions are effective at handling large-scale problems.
+
+### Biology is suited for machine learning
```

Comment:
This subheading seems to orient biology and ML awkwardly. I think we want to emphasize that ML is an appropriate approach for many biological questions. Otherwise, it makes it seem like you're trying to plan a wedding for bio and ML?

I'd also take it a step further and say biomedical data in particular can benefit from ML